uktrade/data-hub-frontend
# Show Aventri Attendees on Event Attendees List page

**Codecov/patch tests are flagging controllers not being tested. These are all going in a separate PR.**
## Description of change

This PR creates a new route `events/aventri/:aventriEventId/attendees` to a new Events attendees page for Aventri events. The page shows a list of names of people who attended an Aventri event, with the data coming from Activity Stream. 

At the moment, what is being displayed are the “Contact First Name” and “Contact Last Name” fields concatenated together. Other fields will be added in later PRs. 

## Test instructions

- In Django dev API, add the user feature flag user-event-activities to your adviser profile.
- Go to the new [Events attendees page](http://localhost:3000/events/aventri/1111/attendees?featureTesting=user-event-activities).
- You should see 2 attendees displayed. 
- You can change the `:aventriEventId` in the url to '2222' or '3333' to see other Aventri event test data. 

## Screenshots
### Before
No before screenshot as this is a new route.

### After
<img width="836" alt="image" src="https://user-images.githubusercontent.com/70902973/176902202-5e8061e4-7f8f-4d6f-ad8a-a5f292d4cccf.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
